### DISCUSSION TITLE
Process for adding a new version of the configuration?

### DISCUSSION THREAD

**Discussion by pulse-mind** [ANXS/postgresql#532] [General]
Hi, 

What is your process to add a new configuration file for a new version, for example for the v14 ?
Do you get the file from https://raw.githubusercontent.com/postgres/postgres/REL_14_STABLE/src/backend/utils/misc/postgresql.conf.sample ?

I've seen you commit the original file, so this file I guess ? 
And then how do you manage the modifications to have a template ? 

Best regards
Fred

**Comment by pulse-mind** [comment]
But I've seen that someone already did the job and create a pull request : https://github.com/ANXS/postgresql/pull/527
when do you plan do merge it ?

**Comment by otakup0pe** [comment]
> What is your process to add a new configuration file for a new version, for example for the v14 ?

I haven't spent a bunch of time in the postgres role, so can't really speak to that.

> when do you plan do merge it ?

when someone gets the tests passing in fedora

**Reply by pulse-mind** [reply]
Or we can write in README that it is not compatible with fedora and if someone needs it he may figure out how to solve the tests?
Fedora is used by a very small percentage. Even more this is an old version that probably few person are going to use with Ansible.

**Comment by pulse-mind** [comment]
I tried many stuff.
* First I run it on a linux VM and the molecule were working fine, except for some versions... let's keep that for later
* but when github workflow is acting we get this error even on ubuntu remote host : 
 ``fatal: [postgresql-13]: UNREACHABLE! => {"changed": false, "msg": "Failed to create temporary directory. In some cases, you may have been able to authenticate and did not have permissions on the target directory. Consider changing the remote tmp path in ansible.cfg to a path rooted in \"/tmp\", for more error information use -vvv. Failed command was: ( umask 77 && mkdir -p \"` echo ~/.ansible/tmp `\"&& mkdir \"` echo ~/.ansible/tmp/ansible-tmp-1670864111.93603-3108-39002375347716 `\" && echo ansible-tmp-1670864111.93603-3108-39002375347716=\"` echo ~/.ansible/tmp/ansible-tmp-1670864111.93603-3108-39002375347716 `\" ), exited with result 1", "unreachable": true}``
I tried to define ANSIBLE_REMOTE_TEMP: /tmp/ansible,... but it is not better. 
I do not understand...

**Comment by MrMegaNova** [comment]
I close this discussion as we have added support for PG 14 & 15 with drop support of older versions / OSs.
